source/common/upstream/load_balancer_impl.h

```diff
@@ -450,26 +455,71 @@ class LeastRequestLoadBalancer : public EdfLoadBalancerBase {
         choice_count_(
             least_request_config.has_value()
                 ? PROTOBUF_GET_WRAPPED_OR_DEFAULT(least_request_config.value(), choice_count, 2)
-                : 2) {
+                : 2),
+        active_request_bias_runtime_(
+            least_request_config.has_value() && least_request_config->has_active_request_bias()
```

Comment:
also test for the value differing from the default?

Reply:
Do you mean we should initialize `active_request_bias_runtime_` with `nullptr` if `least_request_config->active_request_bias()->default_value() == 1.0`?

This would prevent users from making the bias overridable via runtime while still defaulting it to `1.0`.

Reply:
DIdn't mean to limit functionality. I'm confused though if the StatsIntegrationTest flow (a) creates this LB, which is not the default, and (b) overrides this new field. Otherwise I'd expect the overhead to be no more than 64 bytes. How did we get 256 byte overhead per cluster with the default setup?

Reply:
Sorry, got confused. I think this extra optional field, stored as a unique pointer should only cost 8 bytes per cluster if it's being used. DO we think it's worth investigating why the overhead is 256 per cluster? Maybe we'll decide it's needed but it'd be great to understand why.

Reply:
The overhead was 256 byes before switching to unique_ptr. Now it's 8 bytes.

Reply:
github is giving me a confusing view; from https://github.com/envoyproxy/envoy/pull/11252/files this is what I see at the bottom of that web-page:

Left-hand-side: ` EXPECT_MEMORY_EQ(m_per_cluster, 36603);`
right-hand side: ` EXPECT_MEMORY_EQ(m_per_cluster, 36859);`

Do you see something different?

Reply:
The overhead is still 256 bytes =/.

I just tried making `LeastRequestLoadBalancer` use `ENVOY_LOG_MISC` and not inherit from `Logger::Loggable`, but it didn't lead to any memory savings on macOS.

Reply:
So my suspicion is that it's actually creating the optional object for some reason. Maybe it's it worth throwing some log statements or firing up the debugger to check?

Reply:
@jmarantz I reverted the changes to `LeastRquestLoadBalancer` changes but kept the new proto field, and the integration test still complains about those extra 256 bytes.

My guess is that the overhead comes from the protobuf message descriptor, but I wouldn't expect each cluster to hold a copy of it. What do you think?